.github/workflows/pypipublish.yml

```diff
@@ -0,0 +1,29 @@
+name: Upload Python Package on PyPI
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    environment:
+      name: pypi
+      url: https://pypi.org/p/xdggs
+    permissions:
+      id-token: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
```

Comment:
is there a reason why you select python 3.9 specifically? We're building pure-python wheels, so the python version should not matter.

Reply:
No specific reason.. Just copy-pasted from another project :)